src/i18n/lang/dict.es.ts

```diff
@@ -0,0 +1,7 @@
+export const LANG_ES_TRANS = {
```

Comment:
the solution could to apply a complex lines? like "Welcome {name}", where "name" is a variable